src/panels/lovelace/views/hui-view.ts

```diff
@@ -219,16 +226,47 @@ export class HUIView extends LitElement {
       configChanged = !oldLovelace || lovelace.config !== oldLovelace.config;
     }
 
+    let config = lovelace.config.views[this.index!];
+
+    if (!config.badges && !config.cards && (config.area || config.domain)) {
+      if (!subscribedRegistries) {
+        subscribedRegistries = true;
+        subscribeDeviceRegistry(hass.connection, () => undefined);
+        subscribeEntityRegistry(hass.connection, () => undefined);
+      }
+
+      const [deviceEntries, entityEntries] = await Promise.all([
+        subscribeOne(hass.connection, subscribeDeviceRegistry),
```

Comment:
`SubscribeOne` unsubscribes immediately after it gets its first data, you don't want to use that here